tests/bundle/test_bundle_adjustment.py

```diff
@@ -8,12 +8,13 @@
 import dask
 import gtsam
 import numpy as np
-from gtsam import SfmData, SfmTrack
 
+import gtsfm.utils.io as io_utils
 from gtsfm.bundle.bundle_adjustment import BundleAdjustmentOptimizer
+from gtsfm.common.gtsfm_data import GtsfmData
 
 GTSAM_EXAMPLE_FILE = "dubrovnik-3-7-pre"
-EXAMPLE_DATA = gtsam.readBal(gtsam.findExampleDataFile(GTSAM_EXAMPLE_FILE))
+EXAMPLE_DATA = io_utils.read_bal(gtsam.findExampleDataFile(GTSAM_EXAMPLE_FILE))
```

Comment:
don't we need to convert this to `GtsfmData`? It is `SfmData`?

Reply:
The users of gtsfm wont ever need to worry about gtsam's `read_bal`, so I do not think it is an issue.

If you think it needs to be renamed, I'd suggest `read_bal_to_gtsfm_data`.

Reply:
ok, i think its fine as is then